<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BW_CheckWINCC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>BenewakeLogo.ico</normaloff>BenewakeLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>8</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_device">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_device">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>12</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>8</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_channel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>12</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CAN1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CAN2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_channel">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>15</y>
       <width>41</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>通道</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_baud">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>15</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_baud">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>12</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>250kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>33.33kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>66.66kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>83.33kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>125kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>160kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>400kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>666kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>800kbps</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_check">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>341</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>一键执行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_return">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_txID">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>45</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>eg:00 00 00 03</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_canrxID">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN接收ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cantxID">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN发送ID(HEX):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rxID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>580</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_check</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChinese</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actChinese()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnglish</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actEnglish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>trigger_actOpen()</slot>
  <slot>getSerialPort()</slot>
  <slot>refreshSerial()</slot>
  <slot>pollBaudrate()</slot>
  <slot>checkCommunication()</slot>
  <slot>checkAll()</slot>
  <slot>trigger_actChinese()</slot>
  <slot>trigger_actEnglish()</slot>
  <slot>startDevice()</slot>
  <slot>sendCmd()</slot>
  <slot>trigger_actHelp()</slot>
 </slots>
</ui>
